src/graphics/Window.cpp

```diff
@@ -0,0 +1,85 @@
+//
+// Created by ajer on 11.02.19.
+//
+
+#include "Window.h"
+
+
+
+namespace Bald::Graphics {
+
+    Window::Window(const char* title, int width, int height, bool VSync) :
+            m_Title(title),
+            m_Width(width),
+            m_Height(height),
+            m_VSync(VSync) {
+        Window::Init();
```

Comment:
Window::Init() would do exactly the same what Init() and init returns bool it would be smart to check what it returned

Reply:
I would make Init() return void and actually assert when something fails in initialization.